bool_bot/video.py

```diff
@@ -66,6 +66,13 @@ async def video_search(ctx, query):
         "type": 'video'
     }
 
+    if meme:
+        request = {
+            "files": found_files,
+            "message": message,
+            "type": 'video_meme',
```

Comment:
maybe we should create "macros" or some sort of enumeration for the meme types?

Reply:
Yeah I something to rewrite the whole request code. The type is designed to be an enum.

Reply:
Although that will be in another change